libs/openFrameworks/utils/ofUtils.cpp

```diff
@@ -469,10 +454,7 @@ bool ofToBool(const string& boolString) {
 	if(lower == "false") {
 		return false;
 	}
-	bool x = false;
-	istringstream cur(lower);
-	cur >> x;
-	return x;
+	return (bool)std::stoi(boolString);
```

Comment:
I think this was correct. the only case where it'll arrive at that line is if it's a number. any variation in case of false will return earlier.

also there was a problem with oftolower but I fixed it yesterday

El 19 de junio de 2015 8:18:18 CEST, ISHII 2bit <notifications@github.com> escribió:
>> @@ -469,10 +454,7 @@ bool ofToBool(const string& boolString) {
>>  	if(lower == "false") {
>>  		return false;
>>  	}
>> -	bool x = false;
>> -	istringstream cur(lower);
>> -	cur >> x;
>> -	return x;
>> +	return (bool)std::stoi(boolString);
>
>argh...!!
>i think `return (bool)std::stoi(lower);`, but i wrote
>`(bool)std::stoi(boolString);`...
>
>and now, `ofToLower` is wrong...?
>i'll check about `ofToLower`.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/openframeworks/openFrameworks/pull/3948/files#r32803823

-- 
Sent from touchscreen